### DISCUSSION TITLE
Welcome to cherry-studio Discussions!

### DISCUSSION THREAD

**Discussion by kangfenmao** [CherryHQ/cherry-studio#74] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tietie-9587** [comment]
<img width="1352" height="572" alt="image" src="https://github.com/user-attachments/assets/cdec0869-891a-4271-a29a-e716d9d185e8" />
有人知道，模型一直这个报错，要怎么调整吗

**Comment by Tietie-9587** [comment]
亲，你发的东西我已经收到了
